apis/cache/v1beta1/cloudmemorystore_instance_types.go

```diff
@@ -195,6 +226,13 @@ type CloudMemorystoreInstanceObservation struct {
 	// for a given instance so should be checked before each import/export
 	// operation.
 	PersistenceIAMIdentity string `json:"persistenceIamIdentity,omitempty"`
+
+	// Status of in tranit encyption mode the redis service is configured with
+	TransitEncryptionMode string `json:"transitEncryptionMode,omitempty"`
+
+	// ServerCaCerts: Output only. List of server CA certificates for the
+	// instance.
+	ServerCaCerts []*ServerCACertsObservation `json:"serverCaCerts,omitempty"`
```

Comment:
Generally, we don't use pointers in the `Observation` struct. If there isn't a specific reason for it, I would recommend changing to:

```suggestion
	// ServerCaCerts: Output only. List of server CA certificates for the
	// instance.
	ServerCaCerts []ServerCACertsObservation `json:"serverCaCerts,omitempty"`
```